libcxx/include/__config_site.in

```diff
@@ -11,26 +11,27 @@
 
 #cmakedefine _LIBCPP_ABI_VERSION @_LIBCPP_ABI_VERSION@
 #cmakedefine _LIBCPP_ABI_NAMESPACE @_LIBCPP_ABI_NAMESPACE@
-#cmakedefine _LIBCPP_ABI_FORCE_ITANIUM
-#cmakedefine _LIBCPP_ABI_FORCE_MICROSOFT
-#cmakedefine _LIBCPP_HAS_NO_THREADS
-#cmakedefine _LIBCPP_HAS_NO_MONOTONIC_CLOCK
-#cmakedefine _LIBCPP_HAS_NO_TERMINAL
-#cmakedefine _LIBCPP_HAS_MUSL_LIBC
-#cmakedefine _LIBCPP_HAS_THREAD_API_PTHREAD
-#cmakedefine _LIBCPP_HAS_THREAD_API_EXTERNAL
-#cmakedefine _LIBCPP_HAS_THREAD_API_WIN32
+#cmakedefine01 _LIBCPP_ABI_FORCE_ITANIUM
+#cmakedefine01 _LIBCPP_ABI_FORCE_MICROSOFT
+#cmakedefine01 _LIBCPP_HAS_THREADS
+#cmakedefine01 _LIBCPP_HAS_MONOTONIC_CLOCK
+#cmakedefine01 _LIBCPP_HAS_TERMINAL
+#cmakedefine01 _LIBCPP_HAS_MUSL_LIBC
+#cmakedefine01 _LIBCPP_HAS_THREAD_API_PTHREAD
+#cmakedefine01 _LIBCPP_HAS_THREAD_API_EXTERNAL
+#cmakedefine01 _LIBCPP_HAS_THREAD_API_WIN32
+#define _LIBCPP_HAS_THREAD_API_C11 0 // FIXME: Is this guarding dead code?
 #cmakedefine _LIBCPP_DISABLE_VISIBILITY_ANNOTATIONS
-#cmakedefine _LIBCPP_HAS_NO_VENDOR_AVAILABILITY_ANNOTATIONS
+#cmakedefine01 _LIBCPP_HAS_VENDOR_AVAILABILITY_ANNOTATIONS
 #cmakedefine _LIBCPP_NO_VCRUNTIME
```

Comment:
Why is there a few macros that aren't migrated over to 0-1?

Reply:
I'm not quite sure what `_LIBCPP_NO_VSRUNTIME` is. We don't ever define it in `CMakeLists.txt` and there seem to be barely any uses in the wild: https://sourcegraph.com/search?q=context:global+_LIBCPP_NO_VCRUNTIME+-file:libcxx/CMakeLists.txt+-file:__config+-file:new+-file:exception+-file:features.py+-file:libcxx/include/BUILD.gn+-file:typeinfo+-file:libcxx/test/config.py+-file:UsingLibcxx.rst&patternType=keyword&sm=0. We should probably just remove it, but that's a different patch.

`_LIBCPP_HAS_NO_STD_MODULES` seems to be dead. We should just remove it in a separate patch.

I've missed `_LIBCPP_INSTRUMENTED_WITH_ASAN`.